### PR TITLE
Missed import for sys

### PR DESCRIPTION
**Pull request by sjrl**
## Changelog description
Added missing import to sys

## Status

- [x] Code base linted
- [x] Ready to go

### CODE REVIEW

File: qcengine/programs/tests/test_standard_suite.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 
 import pytest
 import qcelemental as qcel
+import sys
```
Comment (loriab) [inline qcengine/programs/tests/test_standard_suite.py:36]: ah yeah, I want that fn to fail at test-design time, and it fails equally well with "no sys" as "sys.exit". thanks.

Comment (sjrl) [reply qcengine/programs/tests/test_standard_suite.py:36]: no problem!

**Comment by sjrl** [comment]
I'm not sure what I need to do for Travis. It looks like there are issues unrelated to this PR?

**Comment by loriab** [comment]
please rebase to fix CI

**Comment by loriab** [comment]
Do you want to rebase this, or shall I just incorporate the edits into another PR?

**Pull request merged**